### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by ron666** [select2/select2#2920]
When I'm doing:

```javascript
$('select').select2();
```

I'm getting an undefined is not a function error.
Diving into the code, I found the line that causes this on **src/js/defaults.js**:

```javascript
name = this.get('amdTranslationBase') + name;
```

The website doesn't have any errors since it's using the previous version

**Comment by smzchaudary** [comment]
@ron666: This was fixed by https://github.com/select2/select2/commit/2bdb5dbdae395612791c6339bf6dbc11a5cce0a5.

**Comment by kevin-brown** [comment]
This should be fixed in 4.0.0-beta.2.
